docs/modules/clusters/pages/ucn-dynamic-config.adoc

```diff
@@ -0,0 +1,38 @@
+= Dynamic Configuration API
+:description: You can use the dynamic configuration API to deploy resources from your client. This means that you can configure namespaces and their resources, and replace the resources at runtime for use when a user customization is next instantiated.
```

Comment:
```suggestion
:description: You can use the dynamic configuration API to deploy resources from your client. This means that you can configure namespaces and their resources, and replace the resources at runtime for use when a user customization is next instantiated or executed.
```
Replacing an existing Namespace at runtime will allow the resources to be available on the next execution of a UDF, although instantiation is also key as some UDFs are long-running, so it would only apply to new executions for those.